ext/node/polyfills/_stream.mjs

```diff
@@ -48,7 +48,7 @@ import {
 // generated with
 // $ esbuild --bundle --legal-comments=none --target=es2022 --tree-shaking=true --format=esm .
 // ... then making sure the file uses the existing ext:deno_node stuff instead of bundling it
-const __process$ = { nextTick };
+import __process$ from "node:process";
```

Comment:
Has this been fixed by regenerating the file using `esbuild` or did you apply it manually?

Reply:
Manually

Reply:
So that means that we will need to remember to update this particular callsite to use `import .. from` instead of `const ...`. Can you update the comment above to call it out specifically?

Reply:
Isn't that what the comment says:
```
//  ... then making sure the file uses the existing ext:deno_node stuff instead of bundling it
```

The imports below are not generated by esbuild and have been manually patched in.